packages/orchestration/src/exos/cosmos-interchain-service.js

```diff
@@ -240,20 +240,23 @@ const prepareOrchestrationKit = (
  * @param {Zone} zone
  * @param {VowTools} vowTools
  */
-export const prepareOrchestrationTools = (zone, vowTools) => {
+export const prepareCosmosInterchainService = (zone, vowTools) => {
   const makeChainAccountKit = prepareChainAccountKit(zone, vowTools);
   const makeICQConnectionKit = prepareICQConnectionKit(zone, vowTools);
-  const makeOrchestrationKit = prepareOrchestrationKit(
-    zone,
-    vowTools,
-    makeChainAccountKit,
-    makeICQConnectionKit,
-  );
+  const makeCosmosOrchestrationServiceKit =
+    prepareCosmosOrchestrationServiceKit(
+      zone,
+      vowTools,
+      makeChainAccountKit,
+      makeICQConnectionKit,
+    );
+
+  const makeCosmosInterchainService = initialPowers =>
+    makeCosmosOrchestrationServiceKit(initialPowers).public;
 
-  return harden({ makeOrchestrationKit });
+  return makeCosmosInterchainService;
 };
-harden(prepareOrchestrationTools);
+harden(prepareCosmosInterchainService);
```

Comment:
We might've miscommunicated yesterday, but remind me where "interchain" (meaning x-chain in general) fits into the mix? And where we'd look to add something ethereum interchain accounts?

Is it maybe better to call  `OrchestrationTools` -> `InterchainService`, but keep `OrchestrationKit` -> `CosmosOrchestrationServiceKit` as that particular exo is scoped just to do cosmos things?

Reply:
To close the loop here - I am seeing in the latest changes: 
 - `cosmosInterchainService` (formerly `orchestration`) added to the bootstrap space. This is what contracts will consume to make ICAs and ICQ.  Likely, this is a footnote to most since they will just use `withOrchestration` / `chain-facade` to create accounts and make queries.
 - This approach leaves space to add something like `evmInterchainService`, or anything else we wish to add, in future core-evals.